python-sdk/example_dags/example_load_file.py

```diff
@@ -253,4 +253,17 @@
     )
     # [END load_file_example_19]
 
+    # [START load_file_example_20]
+    aql.load_file(
+        input_file=File(path="ftp://home/foo/upload/", conn_id="ftp_conn", filetype=FileType.CSV),
+        output_table=Table(
+            conn_id=SNOWFLAKE_CONN_ID,
```

Comment:
Also, can we extend an existing load_file test case where we are verifying the table post-creation?

Reply:
yes sure

Reply:
added integration test case